### PR TITLE
DataHighway mainnet westlake

### PR DESCRIPTION
**Pull request by cgroeschel**

### CODE REVIEW

**Review by jacogr — CHANGES_REQUESTED** [review]
Lint failures. `yarn lint` will fix them locally (if auto-fixable)

**Comment by cgroeschel** [comment]
Uhm, unrelated changes broken the tests?

File: packages/apps-config/src/ui/colors.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ const chainSubDAO = 'linear-gradient(50deg, #F20092 0%, #FF4D5D 100%)';
 const chainSubsocial = '#b9018c';
 const chainTrustBase = '#ff43aa';
 const chainUnitv = '#1452F0';
-const chainVln = '#33cc33';
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:67]: Part of the reason the tests fails.

File: packages/apps-config/src/ui/logos/index.ts [hunk 2]
```diff
@@ -145,7 +146,8 @@ export const chainLogos: Record<string, unknown> = [
   ['TrustBase PC1', nodeTrustBase],
   ['uni arts staging network', nodeUniarts],
   ['Unique', chainUnique],
-  ['VLN PC', nodeVln],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:148]: This removal is what made the tests fails.

File: packages/apps-config/src/api/spec/index.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ import uart from './uart';
 import unique from './unique';
 import unitv from './unitv';
 import vln from './vln';
-import vlnrococo from './vln-rococo';
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:72]: Test failres due to this - Cannot find the `VLN-PC` map to `vlnrococo`

Comment (cgroeschel) [reply packages/apps-config/src/api/spec/index.ts:72]: propably happened during conflict resolution from my last upstream pull, guess I need to add these back in or how to resolve?

Comment (cgroeschel) [reply packages/apps-config/src/api/spec/index.ts:72]: added them back

Comment (cgroeschel) [reply packages/apps-config/src/api/spec/index.ts:72]: et voila :smile:

**Comment by cgroeschel** [comment]
et voila :smile:

**Review by jacogr — APPROVED** [review]

**Pull request merged**